content/strategy-goals/strategy/code-graph/code-intelligence/index.md

```diff
@@ -19,8 +19,8 @@ We generate and process rich metadata that powers compiler-accurate code navigat
 
 ### Vision
 
-> In the near term we want Code intelligence to provide seamless, out-of-the-box, precise code navigation for languages that cover 90% of the market usage.
 > We see Code intelligence as the glue that sticks the product together, providing a platform for features from navigation to precise powered searches, compiler-accurate batch changes and insights. We aim to provide support for all widely used languages and for the ones we don’t, provide a platform for any developer to add and test their own indexers.
+> In the near term we want Code intelligence to provide seamless, out-of-the-box, precise code navigation for languages that cover 90% of the market usage.
```

Comment:
This is a great addition. All of these individual vision items need to be converted to start with `-` instead of `>` though because otherwise it all gets merged together in markdown (see image).

![image](https://user-images.githubusercontent.com/3529318/154218779-7f9aee45-ef0c-4015-9944-987df2b181c6.png)